packages/@sanity/core/src/actions/graphql/extractFromSanitySchema.js

```diff
@@ -9,6 +9,7 @@ const Schema = schemaCompiler.default || schemaCompiler
 const skipTypes = ['document', 'reference']
 const allowedJsonTypes = ['object', 'array']
 const disallowedCustomizedMembers = ['object', 'array', 'image', 'file', 'block']
+const nonScalarTypes = ['document', 'object', 'block', 'reference', 'array']
```

Comment:
I think it's probably better to list the scalar types, as those will be more static?

Reply:
Sure, if you give me a list of those types I can update the change.